src/target/llvm/llvm_module.cc

```diff
@@ -252,18 +251,21 @@ class LLVMModuleNode final : public runtime::ModuleNode {
       LOG(FATAL) << "Fail to load module: " << msg;
     }
     std::string target_;
```

Comment:
Better to use other names...

Reply:
This is something confused me as well...I didn't understand why here is a local variable whose name ends with underscore...

Reply:
I changed it to "target_metadata" because it is something from metadata section of a llvm module